apps/typescriptcourse/src/templates/lesson-template.tsx

```diff
@@ -63,52 +59,37 @@ const ExerciseTemplate: React.FC<{
         }
       >
         <ArticleJsonLd
-          url={`${process.env.NEXT_PUBLIC_URL}/${module.slug.current}/${exercise.slug}`}
-          title={exercise.title}
+          url={`${process.env.NEXT_PUBLIC_URL}/${module.slug.current}/${lesson.slug}`}
+          title={lesson.title}
           images={[
-            `https://image.mux.com/${exercise.muxPlaybackId}/thumbnail.png?width=480&height=384&fit_mode=preserve`,
+            `${getBaseUrl()}/api/video-thumb?videoResourceId=${videoResourceId}`,
```

Comment:
this uses a proxy image endpoint to load the mux data since we don't have the playback id at buildtime and this is "seo"